tests/visual.spec.js

```diff
@@ -21,6 +21,16 @@ function testURL(domain, path) {
     // Gets set once React has finished loading
     await page.locator(`body.react-loaded`);
 
+    if (
```

Comment:
Let's put some code comments here to explain why we're doing this, but other than that this looks good.